### PR TITLE
Show different text in title and body for clarity

### PR DESCRIPTION
**Pull request by unode**
Title is required for valid HTML but using the same text as in the body can cause some confusion.

### CODE REVIEW

**Review by tobyhodges — COMMENTED** [review]
This is a great idea but I think I would prefer a shorter title, so the whole thing can be read in the size of a typical browser tab. How about "Page Title" and "Hello World" for the title and body respectively?

File: _episodes/introduction.md [hunk 1]
```diff
@@ -56,10 +56,10 @@ The simplest, valid HTML `Hello world` is:
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Hello World</title>
+    <title>Hello World from title</title>
```
Comment (unode) [inline _episodes/introduction.md]:
```suggestion
    <title>Page title</title>
```

File: _episodes/introduction.md [hunk 2]
```diff
@@ -56,10 +56,10 @@ The simplest, valid HTML `Hello world` is:
...
   </head>
   <body>
-    <p>Hello World</p>
+    <p>Hello World from body</p>
```
Comment (unode) [inline _episodes/introduction.md]:
```suggestion
    <p>Hello World</p>
```

**Review by unode — COMMENTED** [review]
I agree.
Please squash-merge this one after approving

**Pull request merged**